config/e2e.yml

```diff
@@ -0,0 +1,28 @@
+boolTestKey: true
+numberTestKey: 1
+stringTestKey: 'string'
```

Comment:
This should be removed from e2e as this is testing only config (config/testing.yml)

Reply:
This was left over from the test.yml file but I've removed it